Userland/Libraries/LibCore/EventLoopImplementationUnix.cpp

```diff
@@ -421,6 +421,11 @@ void EventLoopManagerUnix::wait_for_events(EventLoopImplementation::PumpMode mod
     if (error_or_marked_fd_count.value() != 0) {
         // Handle file system notifiers by making them normal events.
         for (size_t i = 1; i < thread_data.poll_fds.size(); ++i) {
+            // FIXME: Make the check work under Android
+#ifdef __ANDROID__
+            auto& notifier = *thread_data.notifier_by_index[i];
+            ThreadEventQueue::current().post_event(notifier, make<NotifierActivationEvent>(notifier.fd(), notifier.type()));
```

Comment:
I'm not sure I understand this commit. In the case that we end up using an EventImplementationUnix, we aren't using ALooper at all. We are using our own poll-based event loop here.

Is it possibly the case that we should be using ALooper in all of the helper processes as well?

in any case, if this is the correct hack to make things not freeze up, we should be looking at AK_OS_ANDROID, not `__ANDROID__`

Reply:
Well, it works on my phone. Without the patch it gets into a loop around this section.
Fixed the look up

Reply:
Fair enough. Can you update the commit message to reflect that, and create an issue to revisit this?

Reply:
Ok, I have described what I could saw here: #484 and I updated the commit comment